core/src/main/java/nl/nn/adapterframework/doc/model/FrankDocGroup.java

```diff
@@ -1,13 +1,113 @@
+/* 
+Copyright 2020 WeAreFrank! 
+
+Licensed under the Apache License, Version 2.0 (the "License"); 
+you may not use this file except in compliance with the License. 
+You may obtain a copy of the License at 
+
+    http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software 
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+
 package nl.nn.adapterframework.doc.model;
 
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 import lombok.Getter;
 
-public class FrankDocGroup {
+/**
+ * Holds a group of FrankElement objects for the Frank!Doc website.
+ * This website will show a table-of-contents with the groups. When you
+ * select a group, the FrankElement objects inside will be shown.
+ * <p>
+ * There are two kinds of groups that are represented by two subclasses of
+ * {@link FrankDocGroup} that are also inner classes, namely
+ * <code>FromType</code> and <code>Other</code>. <code>FromType</code>
+ * holds a group that has the classes of an {@link ElementType}.
+ * <code>Other</code> holds a group of remaining {@link FrankElement}
+ * that can be part of different {@link ElementType} objects.
```

Comment:
'that are each defined with a class rather then with an interface'

Reply:
Gedaan.